packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/index.js

```diff
@@ -18,7 +23,12 @@ const handleCustomScalars = field => {
   if (fieldTypeIsACustomScalar) {
     // if this field is an unsupported custom scalar,
     // type it as JSON
-    field.type.name = `JSON`
+    return {
+      ...field,
+      type: new GraphQLScalarType({
+        name: `JSON`,
+      }),
+    }
```

Comment:
`field.type`, `type.name` etc are now immutable in graphql.js, so these were throwing errors in tests.